### ISSUE TITLE
Support for node v18

### ISSUE THREAD

**Issue by Shinigami92** [davidje13/superwstest#15]
I tried to upgrade my project to node v18, but this line

https://github.com/davidje13/superwstest/blob/71bfd560e1b7796b91622076aa6d410effd02e7a/src/superwstest.mjs#L408

throws `TypeError: address.family.toLowerCase is not a function`

**Comment by davidje13** [comment]
hmm, looks like [this is the culprit](https://github.com/nodejs/node/pull/41431#discussion_r841272416)

Can't say I'm convinced the new version is a better API, but should be easy enough to patch. I'll take a look at it tomorrow.

Something like this should probably do it, but I'll think about whether it can be made a bit cleaner before making the change.

```js
const { family } = address;
if (family === 6 || (typeof family === 'string' && family.toLowerCase() === 'ipv6')) {
```

**Comment by davidje13** [comment]
Published version 2.0.2 with the fix, and added Node 18 to the CI pipeline (doesn't work in GitLab because the docker image for 18 hasn't been published yet, but works on GitHub Actions already).

Since all the tests are passing against 18 in the CI, I think this should be fine now, but feel free to reopen if you find any other issues with using Node 18.

**Comment by Shinigami92** [comment]
I can confirm it works ❤️ 
I also need to wait on GitLab CI docker images and `@types/node`, I just look which of all of my dependencies are compatible and which not 🙂 living bleeding edge 😄
